src/Swashbuckle.AspNetCore.ReDoc/index.js

```diff
@@ -0,0 +1 @@
+Redoc.init('%(SpecUrl)', JSON.parse('%(ConfigObject)'), document.getElementById('redoc-container'));
```

Comment:
There don't appear to have been any code changes that replace these templated vars.

Reply:
There have been some changes in the `RespondWithFile` method, the replace itself works regardless